metabase/metabase
# Fix click behavior for links

Fixes https://github.com/metabase/metabase/issues/24756
Reproduces https://github.com/metabase/metabase/pull/24762

How to test:
- Set `IFRAMED = true` here https://github.com/metabase/metabase/blob/4ac2ef97082c0a34c71aef7f8be550910ab828bc/frontend/src/metabase/lib/dom.js#L13
- Open Metabase with a full-app embedding param, e.g. `http://localhost:3000?top_nav=true`
- Open a dashboard with a card 
- Add a click behavior with a link that navigates to a place within the app to a card
- Click on the card
- Make sure the navigation happens and the top nav bar is preserved
